docs/CONTRIBUTING.md

```diff
@@ -1 +1 @@
-Please refer to [the Contribution Guide](https://argoproj.github.io/argo-cd/developer-guide/contributing/)
+Please refer to [the Contribution Guide](https://argo-cd.readthedocs.io/en/stable/developer-guide/contributing/)
```

Comment:
I think in this case (contributing/developer), we can safely link to `latest` version of the docs.

Reply:
good catch! updated